composer.json

```diff
@@ -24,6 +24,7 @@
         "drupal/core-composer-scaffold": "^10.3",
         "drupal/core-project-message": "^10.3",
         "drupal/core-recommended": "^10.3",
+        "drupal/filefield_paths": "^1.0@beta",
```

Comment:
This currently doesn't meet our requirements for contributed modules:
- Do not use a beta or dev release of a module, even if it provides functionality we are seeking.

I think this might be something to discuss with product, around the risks and benefits.

Reply:
Good catch! @eric-gade let's discuss when you get back?